doc/filters/trim.rst

```diff
@@ -22,5 +26,6 @@ Arguments
 ---------
 
 * ``character_mask``: The characters to strip
-
+* ``mode``: The default is to strip from the start and the end ('both'), but 'left'
+and 'right' will strip from either the left side or right side only.
```

Comment:
And I would enclose `both`, `left` and `right` in double backticks to get proper syntax highlighting in the rendered docs.